astropy/visualization/tests/test_histogram.py

```diff
@@ -67,9 +65,18 @@ def test_histogram_pathological_input():
 
     # The key feature of the data below is that one of the points is very,
     # very different than the rest. That leads to a large number of bins.
-    data = [9.99999914e+05, -8.31312483e-03, 6.52755852e-02, 1.43104653e-03,
-            -2.26311017e-02, 2.82660007e-03, 1.80307521e-02, 9.26294279e-03,
-            5.06606026e-02, 2.05418011e-03]
+    data = [
+        9.99999914e05,
+        -8.31312483e-03,
+        6.52755852e-02,
+        1.43104653e-03,
+        -2.26311017e-02,
+        2.82660007e-03,
+        1.80307521e-02,
+        9.26294279e-03,
+        5.06606026e-02,
+        2.05418011e-03,
+    ]
```

Comment:
This is probably short enough that it doesn't really matter that it is expanded.